jest.config.base.js

```diff
@@ -22,6 +23,5 @@ export default {
     // Ignore '.js' at the end of imports; part of ESM support.
     '^(\\.{1,2}/.*)\\.js$': '$1',
   },
-  // this can be removed with jest v29
-  snapshotFormat: { escapeString: false, printBasicPrototype: false },
+  prettierPath: createRequire(import.meta.url).resolve('prettier-2'),
```

Comment:
Jest v29 isn't friends with prettier v3 yet, this is the recommended fix in the jest docs for now.

Reply:
dunno why this is in this PR rather than renovate or something but 🤷

Reply:
I can separate, but this is the first time writing a new snapshot since taking prettier@3 and it yelled at me. Not sure how renovate would make this change.

Reply:
Ah gotcha, that's why it's related.